common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -144,6 +151,23 @@ def create_pull_request(repo, pr_name, source_branch, target_branch,
       end
       # rubocop:enable Metrics/ParameterLists
 
+      def decline_pull_request(repo, pr_id, comment = nil)
+        # https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/
```

Comment:
Do you want to deep link to https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/#api-repositories-workspace-repo-slug-pullrequests-pull-request-id-decline-post ?

Either way is fine with me, shorter is also less likely to break.

Reply:
I choose just the page to keep it shorter.
And also to give a possible reference to other pull request related calls.
So I'd keep it like it is.